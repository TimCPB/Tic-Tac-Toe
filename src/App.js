import React, { Component } from "react";
import "./App.css";

class App extends Component {
  render() {
    return(
      <div classname="App">
        <h1>Yo world!</h1>
      </div>
    );
  }
}

export default App;