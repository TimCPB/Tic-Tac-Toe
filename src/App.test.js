import React from 'react';
import App from './App.react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Appp page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});