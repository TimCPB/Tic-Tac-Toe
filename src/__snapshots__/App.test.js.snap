// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    Yo world! I'm using native React
  </h1>
</div>
`;
